tests/lib/rules/no-callback-leaks-in-ember-objects.js

```diff
@@ -0,0 +1,152 @@
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-callback-leaks-in-ember-objects');
+const RuleTester = require('eslint').RuleTester;
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+
+const message = "Event listeners and interval timers must be unregistered or ensure that the context they're registered with is destroyed, when no longer needed.";
```

Comment:
This error message should be imported directly from the rule file instead of copied here. See the `no-test-and-then` rule for an example.

Reply:
Done